setup.cfg

```diff
@@ -32,10 +32,10 @@ include_package_data = True
 python_requires = >=3.6
 test_suite =
 install_requires =
+    pyxdg>=0.26
     requests>=2.24
     toml>=0.10.2
     urllib3>=1.24.2
-    pyxdg>=0.26
```

Comment:
Ugh...really? Please tell me this doesn't need to also match `requirements.txt` and `pyproject.toml`?

If so...same 'add a comment' suggestion should apply here too.

-*sigh*-